ant-evm/src/data_payments.rs

```diff
@@ -17,7 +17,7 @@ pub use std::time::SystemTime;
 use xor_name::XorName;
 
 /// The time in seconds that a quote is valid for
-pub const QUOTE_EXPIRATION_SECS: u64 = 3600;
+pub const QUOTE_EXPIRATION_SECS: u64 = 3600 * 24;
```

Comment:
changing the quote expiration will have certain impact on the backward compartibility side.
especially against the mainnet.
The value itself may also need to be further discussed.

Reply:
Indeed, quoting will still be limited to 1 hour on older nodes while newer nodes will be more patient. 
With time, the network as a whole will be upgraded. 
In the meantime we can implement parallel fixes to better support larger file uploads.

Reply:
As David suggested to separate payment and upload, this quoting expiration doesn't need to be extened.
I'd suggest remain this as it is to avoid extra hassle.

Reply:
Ok make sense, will revert to 1 hour then.